Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN adduser --disabled-password --gecos '' deploy
 RUN mkdir -p /src
 RUN chown deploy:deploy /src
 
-RUN pip --no-cache-dir install poetry
+RUN pip --no-cache-dir install poetry==1.8.0
```

Comment:
`poetry` [released ](https://pypi.org/project/poetry/#history)`2.0.0` on Jan 5th and it breaks the build.